snapcraft.yaml

```diff
@@ -113,18 +114,19 @@ parts:
     - pbzip2
     - python-pip
     - python2.7-dev
-    - python3.5-dev
     - python3.6-dev
     - python3.7-dev
     - python3.8-dev
     - swig
+    - software-properties-common
```

Comment:
This package is installed when running the `linux_optional_compilation_dependencies.sh`.
Is it really required to add it to the `build-packages` list?

Reply:
Probably not. I think I added it when pursuing a different solution that required it and then didn't notice it wouldn't be required when I switched plans.

Reply:
Removed in latest commit.